packages/next-app/src/components/OwnedPool.tsx

```diff
@@ -2,23 +2,46 @@ import { Badge, Button, HStack, Text, VStack } from "@chakra-ui/react";
 
 interface IOwnedPool {
   onClick: () => void;
-  address: string;
-  type: string;
-  ratio?: string;
-  name: string;
+  address?: string;
+  poolType?: string | null | undefined;
```

Comment:
no need to write `poolType?: ... | undefined` -- https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters